### PR TITLE
Improve confine-mouse-mode PR (take 2)

### PR DESCRIPTION
**Pull request by copygirl**

### CODE REVIEW

File: src/OpenTK/Platform/Linux/LinuxNativeWindow.cs [hunk 1]
```diff
@@ -512,6 +500,17 @@ public override Size ClientSize
             }
         }
 
+        public override bool CursorGrabbed
+        {
```
Comment (VPeruS) [inline src/OpenTK/Platform/Linux/LinuxNativeWindow.cs]: Frassle already denied such change

Comment (copygirl) [reply src/OpenTK/Platform/Linux/LinuxNativeWindow.cs]: What exactly? Moving the Property, or removing the `NotImplementedException`?

Comment (VPeruS) [reply src/OpenTK/Platform/Linux/LinuxNativeWindow.cs]: removing the NotImplementedException

**Comment by copygirl** [comment]
Removed the commit removing the `NotImplementedException`.

**Comment by VPeruS** [comment]
👍

**Pull request merged**